pkgs/by-name/nt/ntfs2btrfs/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, cmake, fmt, lzo, pkg-config, zlib, zstd }:
+
+stdenv.mkDerivation rec {
+  pname = "ntfs2btrfs";
+  version = "20240115";
+
+  nativeBuildInputs = [ cmake fmt lzo pkg-config zlib zstd ];
+
+  src = fetchFromGitHub {
+    owner = "maharmstone";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-sZ8AWREe2jasy3hqLTjaLcOMCNsrDjz2eIuknA2TsEs=";
+  };
+
+  meta = {
+    description = "A CLI tool which does in-place conversion of Microsoft's NTFS filesystem to the open-source filesystem Btrfs";
+    homepage = "https://github.com/maharmstone/ntfs2btrfs";
+    license = with lib.licenses; [ gpl2Only ];
+    maintainers = with lib.maintainers; [ j1nxie ];
+    mainProgram = "ntfs2btrfs";
+    platforms = lib.platforms.all;
+    badPlatforms = [ "x86_64-darwin" "i686-darwin" "aarch64-darwin" "arm7a-darwin" ];
```

Comment:
```suggestion
    platforms = lib.platforms.linux;
```
this is cleaner

Reply:
fair enough, i was just going off what the docs were suggesting, though i do agree this looks cleaner.